pretix_sepadebit/payment.py

```diff
@@ -114,9 +137,45 @@ def settings_form_fields(self):
                                  'configuring at least 7 days.'),
                      min_value=1
                  )),
+                ('earliest_due_date',
+                 forms.DateField(
+                     label=_('Earliest debit due date'),
+                     help_text=_('Earliest date the direct debit can be due. '
+                                 'This date is used as the direct debit due date if the order date plus pre-notification time would result in a due date earlier than this. '
+                                 'Customers with orders using the earliest due date will receive an email reminding them about the upcoming charge based on the configured pre-notification days.'),
+                     required=False,
+                     widget=forms.widgets.DateInput(attrs={'class': 'datepickerfield'})
+                 )),
+                ('mail_payment_reminder_subject',
+                 I18nFormField(
+                     label=_("Due date reminder subject"),
```

Comment:
It appears to me that the whole banking industry refers to this as the pre-notification, maybe we should also call it "Pre-notification mail subject" and not a reminder? This is really nitpicky though.